srcCpp/RTIDDSImpl.h

```diff
@@ -95,6 +96,7 @@ class RTIDDSImpl : public IMessaging
         _typename = T::TypeSupport::get_type_name();
 
         _pongSemaphore = NULL;
+        _loggerDevice = RTIDDSLoggerDevice();
```

Comment:
Move it to the beginnning of the function:

```
     RTIDDSImpl() :
         _transport(),
         _loggerDevice()
     {
```

Reply:
Done